helm_deploy/digital-prison-services/values.yaml

```diff
@@ -1,12 +1,69 @@
 ---
 # Values here are the same across all environments
+generic-service:
+  nameOverride: digital-prison-services
 
+  image:
+    repository: quay.io/hmpps/digital-prison-services
+    tag: latest
+    pullPolicy: IfNotPresent
+    port: 3000
+
+  ingress:
+    enabled: true
+    v1_2_enabled: true
+    v0_47_enabled: false
+    host: app-hostname.local    # override per environment
+    tlsSecretName: digital-prison-services-cert
+    contextColour: green
```

Comment:
I think this has been dropped from the `generic-service` now that the _live-1_ to _live_ migration has been completed?